sections/telemetry_system.tex

```diff
@@ -298,6 +313,38 @@ \subsection{Scheduling}
 The telemetry system will immediately perform boot operations to load all the device drivers before launching into the
 application code at power-on.
 
+\section{Data Buffers}
```

Comment:
This is a great explanation! If you have time to make a diagram like the one you showed me on the whiteboard that might help to visualize!

Reply:
I've got a draft of one in the PDR folder that I just made. I'm not sure if I should bother showing the operations on it or just leave it a hint at what the verbal or written explanation means

Reply:
It looks good! You can show the operations if you want, the more detail the better!